### PR TITLE
Add PHP version to the Symphony Overview

### PR DESCRIPTION
**Pull request by tiloschroeder**
- Add the PHP version to the "Symphony Overview" panel

### CODE REVIEW

**Review by nitriques — APPROVED** [review]
Hey ! Thanks for this!

File: extension.driver.php [hunk 1]
```diff
@@ -519,7 +519,21 @@ public function render_panel($context) {
...
+				$version = phpversion();
+				if ($ver > 7.0) {
+					$dl->appendChild(new XMLElement('dt', __('PHP Version')));
+					$dl->appendChild(new XMLElement('dd', '<span style="color:green">' . $version . '</span>'));
+				} elseif ($ver === 7.0) {
+					$dl->appendChild(new XMLElement('dt', __('PHP Version')));
+					$dl->appendChild(new XMLElement('dd', '<span style="color:orange">' . $version . '<br />Please go to your server management and check if a newer PHP version is available.</span>'));
+				} else {
+					$dl->appendChild(new XMLElement('dt', __('PHP Version')));
+					$dl->appendChild(new XMLElement('dd', '<span style="color:red">' . $version . '<br />Your PHP version is outdated. For security reasons, please go to your server management and set the current PHP version (> 7.0) for this host.</        span>'));
```
Comment (michael-e) [inline extension.driver.php:534]: There are some superfluous space characters in the closing span tag.

More important: We shouldn't introduce inline styles here. The extension has its own CSS file, so this is where additional styles should be located.

**Review by michael-e — COMMENTED** [review]
I commented on the code. I strongly suggest to change this.

File: extension.driver.php (hunk 1 above)
Comment (tiloschroeder) [reply extension.driver.php:534]: You're right, @michael-e. I'll send a second PR.

**Pull request merged**